### DISCUSSION TITLE
browserlist - outdated?

### DISCUSSION THREAD

**Discussion by topvis** [quasarframework/quasar#11017] [General - Components / Directives / etc] [answered]
I have this in m package.json:
`
  "browserslist": [
    "last 1 Chrome versions"
  ],
`
quasar dev shows "Chrome >= 91" when compiling codes. Current latest stable Chrome is version 94. Seems browserlist is outdated. Should I install browserlist in the project?

By the way, if I set the version directly in package.json like below, it won't be accepted by quasar dev.
`
  "browserslist": [
    "Chrome >= 94"
  ],
`

**Comment by hawkeye64** [comment, marked as answer]
Edit: updated to be marked as the answer...

I think you can do something like this: `npx browserslist@latest --update-db`
run `npm update` or `yarn upgrade` _this command updates the dependencies of the libraries._
now quasar dev shows the updated browser versions.

**Comment by topvis** [comment]
Thank you @hawkeye64. I ran `npx browserslist@latest --update-db` and it does updated browserslist's database. Now I run npx browserslist and it shows the latest version `chrome 94` correctly. But quasar dev still shows "Chrome >= 91" when compiling.

**Reply by hawkeye64** [reply]
maybe try a `quasar clean` in case it's being cached somewhere

**Reply by topvis** [reply]
I tried `quasar clean` and it didn't resolve my issue. I found the solution from here: https://github.com/browserslist/browserslist/issues/492

two steps:
1. run `npx browserslist@latest --update-db`
2. run `npm update` or `yarn upgrade`   _this command updates the dependencies of the libraries._
now `quasar dev` shows the updated browser versions.
